ibis/sql/postgres/api.py

```diff
@@ -90,6 +91,7 @@ def connect(
     --------
     >>> import os
     >>> import getpass
+    >>> from ibis.sql.postgres.api import connect
```

Comment:
Can you remove this and change the `connect(...)` line to `ibis.postgres.connect`? That's the correct user-facing API.

Reply:
@cpcloud Okay. I'll follow up in next monday. Thanks.

Reply:
@cpcloud Sorry for late. I updated it. I'll look on build fail issue, too.